_maps/RandomRuins/IceRuins/bubberstation/icemoon_interdyne.dmm

```diff
@@ -5685,8 +5685,6 @@
 /area/ruin/syndicate_lava_base/spare)
 "OD" = (
 /obj/structure/frame/machine/secured,
-/obj/item/circuitboard/machine/ammo_workbench,
```

Comment:
Ammo workbench should be separate so we can discuss it separately

Reply:
Sure